### ISSUE TITLE
Not supposed to use `net5-windows` apparently

### ISSUE THREAD

**Issue by Deadpikle** [punker76/gong-wpf-dragdrop#371]
RE: .NET 5 compatibility:

Just saw this on Twitter today [here](https://twitter.com/clairernovotny/status/1329472367781945344) by a .NET PM.

> Pro Tip: DO NOT USE the `net5.0-windows` TFM without specifying the SDK version. You should be using `net5.0-windows10.0.19041` and the like. 
> That's the SDK and YOU CAN target and run on earlier Windows versions.

I'm not sure which Windows version should be referenced, here, so I've not submitted a PR or anything. The twitter thread linked above has some more info. I guess the earliest OS this works on...? I don't know.

**Comment by Deadpikle** [comment]
I'm closing this as "not an issue" for a few reasons:

1. The Twitter thread is not clear on what should be done. https://twitter.com/terrajobst/status/1329480198404722688?s=20 is from another .NET PM and he says `net5-windows` is just fine.
2. System.Reactive has the following in their csproj for target frameworks: `net5.0;net5.0-windows10.0.19041` (thus targeting both). But, if I do something similar in this project, there are warnings in the Packages portion of the csproj file listing.
3. I don't think Gong WPF uses any WinRT APIs, so do we really need to manually update the SDK version this is built with every time Windows releases something new? I don't think so.

tl;dr: I think things are just fine.
